core/src/main/java/com/linecorp/armeria/common/stream/CancellableStreamMessage.java

```diff
@@ -42,9 +42,9 @@
 
 import io.netty.util.concurrent.EventExecutor;
 
-abstract class AbstractStreamMessage<T> implements StreamMessage<T> {
+abstract class CancellableStreamMessage<T> extends AggregationSupport implements StreamMessage<T> {
```

Comment:
Is there any particular reason that you want to rename this class?

Reply:
I thought `AbstractStreamMessage` is a too general name for this class because it is only used for `DeferredStreamMessage` and `DefaultStreamMessage`.
But I'm not strong here. It is an internal class name so we can change it later.

Reply:
SGTM. 👍